alpha/apps/kaltura/lib/myKuserUtils.class.php

```diff
@@ -2,6 +2,7 @@
 
 class myKuserUtils
 {
+	const SPECIAL_CHARS = array('+', '=', '-', '@', ',');
```

Comment:
const with array is not supported in low php versions
better just put it in the function